src/modules/rlm_python/rlm_python.c

```diff
@@ -205,6 +206,58 @@ static void mod_error(void)
 	Py_XDECREF(pTraceback);
 }
 
+static int dlopen_linked_libpython_cb(struct dl_phdr_info *info,
+					UNUSED size_t size, void *data)
+{
+	const char *pattern = "/libpython"
+				STRINGIFY(PY_MAJOR_VERSION) "."
+				STRINGIFY(PY_MINOR_VERSION) ".so";
```

Comment:
Well, regarding path matching, the filename is still the same as before the patch (e.g. "libpython2.7.so"), only the slash is added, which should be the same on all platforms FreeRADIUS targets, AFAIK.

I'm not sure about dl_iterate_phdr, but it seems it is widely supported.